h2o-automl/src/main/java/ai/h2o/automl/modeling/StackedEnsembleStepsProvider.java

```diff
@@ -376,6 +390,11 @@ protected void setMetalearnerParameters(StackedEnsembleParameters params) {
                         }
                     });
                     optionalSeSteps.add(new AllSEModelStep("all_xglm", optionalGroup, aml()) {
+                        @Override
+                        protected boolean hasDoppelganger(Key<Model>[] keys) {
+                            return false;
```

Comment:
Can't we do better? Something like `number of base models > number of "best of family" models`. So we don't end up with `all_xglm == best_of_family_xglm`.

Reply:
good point, will see how I can improve this: wanted to keep the logic simple for this one.